content/hi/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "माइक्रोप्रोफाइल, कॉन्फिगमैप्स और सीक्रेट्स का उपयोग करके कॉन्फिगरेशन को लागू करना"
+content_type: ट्यूटोरियल
+weight: 10
+---
+
+<!-- overview -->
+
+इस ट्यूटोरियल में आप सीखेंगे कि अपने माइक्रोसर्विस के कॉन्फ़िगरेशन को कैसे और क्यों लागू करना है। 
+विशेष रूप से, आप सीखेंगे कि एनवायरमेंट वेरिएबल सेट करने के लिए कुबेरनेट्स कॉन्फिगमैप्स और सीक्रेट्स का 
+उपयोग कैसे करें और फिर माइक्रोप्रोफाइल कॉन्फिग का उपयोग करके उनका उपभोग करें। 
+
+
+## {{% heading "prerequisites" %}}
+
+### कुबेरनेट्स कॉन्फ़िगमैप्स और सीक्रेट बनाना
+
+कुबेरनेट्स में डॉकर कंटेनर के लिए एनवायरमेंट वेरिएबल सेट करने के कई तरीके हैं, जिनमें शामिल हैं: डॉकरफाइल, 
+कुबेरनेट्स.वाईएमएल, कुबेरनेट्स कॉन्फिगमैप्स और कुबेरनेट्स सीक्रेट्स। ट्यूटोरियल में, आप सीखेंगे कि अपने 
+एनवायरमेंट वेरिएबल सेट करने के लिए कुबेरनेट्स कॉन्फिगमैप्स और कुबेरनेट्स सीक्रेट्स का उपयोग कैसे करें, जिनके वैल्यू
+आपके माइक्रोसर्विसेज में इंजेक्ट किए जाएंगे। कॉन्फिगमैप्स और सीक्रेट्स का उपयोग करने का एक लाभ यह है कि 
+उन्हें कई कंटेनरों में फिर से उपयोग किया जा सकता है, जिसमें विभिन्न कंटेनरों के लिए अलग-अलग एनवायरमेंट वेरिएबल
```

Comment:
Although this is a literal translation, I would request that you make it more idiomatic. The phrasing is confusing to a normal reader. Right now the object/subject agreement is not being fulfilled for the second part of the sentence.

You have to convey the following, "One of the benefits for using ConfigMaps and Secrets is that they can be re-used across multiple containers, including being assigned to different environment variables for the different containers.:

Reply:
How about: "Docker कंटेनर में कुबेरनेट्स के लिए एनवायरमेंट वेरिएबल सेट करने के कई तरीके हैं, जिनमें शामिल हैं:
डॉकरफाइल, कुबेरनेट्स.वाईएमएल फ़ाइल, कुबेरनेट्स कॉन्फिगमैप्स और कुबेरनेट्स सीक्रेट्स। इस ट्यूटोरियल में, आप कॉन्फिगमैप्स और सीक्रेट्स का उपयोग करके अपने एनवायरमेंट वेरिएबल कैसे सेट करें, यह सीखेंगे। इन एनवायरमेंट वेरिएबल के मान आपकी माइक्रोसर्विसेज़ (छोटी स्वतंत्र सेवाएं) में इंजेक्ट किए जाएंगे। कॉन्फिगमैप्स और सीक्रेट्स का उपयोग करने का एक लाभ यह है कि इन्हें कई कंटेनरों में रीयूज़ किया जा सकता है, जिसमें विभिन्न कंटेनरों के लिए अलग-अलग एनवायरमेंट वेरिएबल असाइन करना शामिल है।" ?

Also, would we prefer keeping "Dockerfile, kubernetes.yml, Kubernetes ConfigMaps, and Kubernetes Secrets" - as it is (not translating to Hindi) ?

Reply:
@anushatomar13  It will looks good one think that I think is good  is adding glossary to some of these will be more useful.

Reply:
> Also, would we prefer keeping "Dockerfile, kubernetes.yml, Kubernetes ConfigMaps, and Kubernetes Secrets" - as it is (not translating to Hindi) ?
@anushatomar13  sounds good.

Reply:
@niranjandarshann Just to be clear, should I change this for the whole page?

Reply:
no no the paragraph is refering to the  reference  of how one can set the environment  variable. I hadn't find any other place where the same refernece is used . So change only for this paragraph.